nix/hydra/packages.nix

```diff
@@ -116,73 +116,69 @@ rec {
 
   hydraw-static = musl64Pkgs.hydraw.components.exes.hydraw;
 
-  tests = {
-    plutus-cbor = pkgs.mkShellNoCC {
-      name = "plutus-cbor-tests";
-      buildInputs = [ nativePkgs.plutus-cbor.components.tests.tests ];
-    };
-    plutus-merkle-tree = pkgs.mkShellNoCC {
-      name = "plutus-merkle-tree-tests";
-      buildInputs = [ nativePkgs.plutus-merkle-tree.components.tests.tests ];
-    };
-    hydra-plutus = pkgs.mkShellNoCC {
-      name = "hydra-plutus-tests";
-      buildInputs = [ nativePkgs.hydra-plutus.components.tests.tests ];
-    };
-    hydra-node = pkgs.mkShellNoCC {
-      name = "hydra-node-tests";
-      buildInputs = [
-        nativePkgs.hydra-node.components.tests.tests
+  plutus-cbor-tests = pkgs.mkShellNoCC {
```

Comment:
What's the reason for renaming (I don't mind, just want to know whether there is a convention or rationale here)

Reply:
`nix flake show` does not support nested attribute sets in packages.${system}. It's true currently that these derivations don't show up in the nix flake show output, but in flake-parts it's also an error.